src/styles/repeatable-set.scss

```diff
@@ -22,7 +22,7 @@
     }
   }
   > .ob-button__add {
-    margin-bottom: 0;
+    margin-bottom: $size-5;
```

Comment:
Should this be conditional based on the type?

Reply:
Ah yeah, addressing